exercises/concept/plane-tickets/.meta/exemplar.py

```diff
@@ -0,0 +1,44 @@
+""" Plane Tickets Exercise """
+
+def generate_seats(amount):
+    
+    """ Generate a series of seat numbers for airline boarding.
+    
+    :param amount: Amount of seats to be generated. (int)
+    :return: Iterable that generates seat numbers. (generator)
+    
+    There should be no row 13
+    
+    Seat numbers are generated with each row having 4 seats.
+    These should be sorted from low to high.
+    
+    Example: 3C, 3D, 4A, 4B
+    
+    """
+    
+    SEATS_IN_ROW = ["A", "B", "C", "D"]
+    
+    amount = amount+4 if amount >= 13 else amount
```

Comment:
Operators like `+` should be space padded.

Reply:
```suggestion
    amount = amount + 4 if amount >= 13 else amount
```
This actually does not throw an error in Pylint, but I do agree with you.